pkg/operator/configobserver/cloudprovider/observe_cloudprovider.go

```diff
@@ -78,12 +61,12 @@ func (c *cloudProviderObserver) ObserveCloudProviderNames(genericListers configo
 	}
 	if err != nil {
 		errs = append(errs, err)
-		return previouslyObservedConfig, errs
+		return existingConfig, errs
```

Comment:
nit could be changed to `return existingConfig, append(errs, err)`

Reply:
done